cmd/controller/state/imagescan/scanner.go

```diff
@@ -423,6 +423,9 @@ func scanJobSpec(
 			Template: corev1.PodTemplateSpec{
 				ObjectMeta: metav1.ObjectMeta{
 					Annotations: annotations,
+					Labels: map[string]string{
+						"azure.workload.identity/use": "true",
```

Comment:
Thinking if we can somehow add it only for azure. Maybe detect that this runs on azure nodes by checking for azure specific labels?

Reply:
Would be nice if this could be controlled via the helm chart to be honest. So maybe add a conditional flag to kvisor, or use some env variable to control it.

Reply:
@anjmao what would be a good way to detect we are running in Azure? the pod does not see it's own annotations and I don't see any azure specific env variables. 
btw you linked this PR in your comment, maybe you meant to link to something else?

Reply:
hmm, we have the `kube-bench-cloud-provider` that we can use, or even add another variable to the setup flow that will indicate the provider